### PR TITLE
Add created_by field to API docs and references

### LINKED ISSUE #2243: Add created_by field to object metadata
### Description
https://github.com/sensu/sensu-go/pull/3604 adds a `created_by` field to object metadata which is populated when resources are created or updated via the HTTP API.

### Category

This is an issue with:

- [ ] errata (error in doc content)
- [ ] new doc (requires a completely new doc)
- [x] update (add missing or refresh existing content)
- [ ] improvement (improve docs content or format)

### Affected pages
I would assume many pages will need to be updated. Here are a few that I can think of off the top of my head:
- Documentation about the object metadata struct
- Reference docs for all resources that contain object metadata
- API docs examples and fields

### Expected behavior
`created_by` is a non-required string indicating the sensu user who created the resource or last updated the resource.

### Current behavior
`created_by` field does not exist, will be introduced in 5.19.0.

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
- Adds `created_by` to metadata object in GET examples in API docs
- Adds `created_by` to metadata and spec tables in references

## Motivation and Context
Closes https://github.com/sensu/sensu-docs/issues/2243

### CODE REVIEW

File: content/sensu-go/5.19/reference/assets.md [hunk 1]
```diff
@@ -505,7 +513,7 @@ spec:
   "api_version": "core/v2",
   "metadata": {
     "name": "check_cpu",
-    "namespace": "default",
+    "namespace": "default"
```
Comment (hillaryfraley) [inline content/sensu-go/5.19/reference/assets.md]:
```suggestion
    "namespace": "default",
```

File: content/sensu-go/5.19/reference/assets.md [hunk 2]
```diff
@@ -429,7 +437,7 @@ spec:
   "api_version": "core/v2",
   "metadata": {
     "name": "check_cpu_linux_amd64",
-    "namespace": "default",
+    "namespace": "default"
```
Comment (hillaryfraley) [inline content/sensu-go/5.19/reference/assets.md]:
```suggestion
    "namespace": "default",
```

File: content/sensu-go/5.19/reference/assets.md [hunk 3]
```diff
@@ -601,7 +610,8 @@ spec:
   "type": "Asset",
   "api_version": "core/v2",
   "metadata": {
-    "name": "sensu-email-handler"
+    "name": "sensu-email-handler",
+    "namespace": default,
```
Comment (hillaryfraley) [inline content/sensu-go/5.19/reference/assets.md]:
```suggestion
    "namespace": default
```

File: content/sensu-go/5.19/reference/assets.md [hunk 4]
```diff
@@ -621,7 +631,7 @@ spec:
   "api_version": "core/v2",
   "metadata": {
     "name": "prometheus_collector",
-    "namespace": "default"
+    "namespace": "default",
```
Comment (hillaryfraley) [inline content/sensu-go/5.19/reference/assets.md]:
```suggestion
    "namespace": "default"
```

File: content/sensu-go/5.19/api/mutators.md [hunk 5]
```diff
@@ -170,6 +173,7 @@ output               | {{< highlight json >}}
   "metadata": {
     "name": "example-mutator",
     "namespace": "default",
+    "created_by" "admin",
```
Comment (hillaryfraley) [inline content/sensu-go/5.19/api/mutators.md]:
```suggestion
    "created_by": "admin",
```

File: content/sensu-go/5.19/api/silenced.md [hunk 6]
```diff
@@ -185,6 +188,7 @@ output               | {{< highlight json >}}
   "metadata": {
     "name": "linux:check-cpu",
     "namespace": "default",
+    "created_by" "admin",
```
Comment (hillaryfraley) [inline content/sensu-go/5.19/api/silenced.md]:
```suggestion
    "created_by": "admin",
```

**Review by nikkictl — APPROVED** [review]
These changes look good. I'm not sure if there are any other areas of the docs that need this update, as it applies to the API only, but LGTM!

**Pull request merged**